proxy/http.go

```diff
@@ -86,8 +86,19 @@ func NewRequestBuilderMiddleware(remote *config.Backend) Middleware {
 		}
 		return func(ctx context.Context, request *Request) (*Response, error) {
 			r := request.Clone()
-			r.GeneratePath(remote.URLPattern)
-			r.Method = remote.Method
+
+			if remote.URLPattern == "/*path" {
+				r.Path = r.Params["Path"]
+			} else {
+				r.GeneratePath(remote.URLPattern)
+			}
```

Comment:
If you want to use a single param `Path` extracted from the original request, you can just define a backend URLPattern like this `/{path}`

Reply:
if i tested this right, it wont work for:
gateway/my_service_alias/{anypath}?params ->
my_service/{anypath}?params

where {anypath} could be "/a" or "/a/b" etc

Reply:
During the design of KrakenD we intentionally avoided using wildcards as this is not considered a best practice for an api gateway. Of course we understand the use case and that it might make sense in some scenarios. 

We really appreciate your first contribution, although this looks like a middlewware rather than a core modification. If you need this feature and you think it could be useful for the community, you can create a custom component with a middleware replacing the standard implementation of the `proxy.NewRequestBuilderMiddleware` and inject it into your `proxy.Factory` (https://github.com/devopsfaith/krakend/blob/master/proxy/factory.go#L89). We'll be happy to add it into the KrakenD-contrib listing and the KrakenD-ce

Thanks for your first contribution!

Cheers!